include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3370,6 +3370,12 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
                                            *this->dof_info);
     }
 
+  const bool is_ecl =
+    (this->dof_access_index ==
+       internal::MatrixFreeFunctions::DoFInfo::dof_access_cell &&
+     this->is_interior_face() == false) ||
+    (!is_face && !this->is_interior_face());
```

Comment:
Shouldn't this be
```suggestion
  const bool is_ecl =
    is_face &&
    this->dof_access_index ==
       internal::MatrixFreeFunctions::DoFInfo::dof_access_cell &&
     this->is_interior_face() == false;
```

Reply:
We need to check this for `face && !is_face` because of `FEEvaluationShift`. I have simplified the condition in 83f02865f7b8b9f210df82cdee62a9cbb838e2de . Let's see what the tests say.